xml/System.Diagnostics/Debugger.xml

```diff
@@ -356,6 +356,7 @@ Console.WriteLine("Hello, world.");
   
 ## Remarks  
  If a debugger is already attached, nothing happens.  
+ Debugger launch is only supported on Windows. On Unix, the method will return true without launching a debugger.
```

Comment:
```suggestion

Debugger launch is only supported on Windows. On Unix operating systems, the method returns `true` without launching a debugger.
```

Reply:
Thanks, applied. BTW in future don't hesitate to apply suggested edits, I'm never going to mind :)